hq/app/schedule/IamListAccessKeys.scala

```diff
@@ -0,0 +1,49 @@
+package schedule
+
+import aws.AwsAsyncHandler.{awsToScala, handleAWSErrs}
+import aws.iam.IAMClient.SOLE_REGION
+import aws.{AwsClient, AwsClients}
+import com.amazonaws.services.identitymanagement.AmazonIdentityManagementAsync
+import com.amazonaws.services.identitymanagement.model.{AccessKeyMetadata, ListAccessKeysRequest, ListAccessKeysResult}
+import model.{AccessKeyWithId, AwsAccount, VulnerableAccessKey, VulnerableUser}
+import utils.attempt.Attempt
+
+import scala.collection.JavaConverters._
+import scala.concurrent.ExecutionContext
+
+object IamListAccessKeys {
+
+  // get information on all access keys for a given AWS account
+  def listAccountAccessKeys(account: AwsAccount, users: Seq[VulnerableUser], iamClients: AwsClients[AmazonIdentityManagementAsync])
+    (implicit ec: ExecutionContext): Attempt[List[VulnerableAccessKey]] = {
+    val accessKeyData = users.map { user =>
+      for {
+        client <- iamClients.get(account, SOLE_REGION)
+        keys <- listAccessKeys(client, user)
+      } yield keys.getAccessKeyMetadata.asScala.toList
+    }.toList
+    Attempt.flatSequence(accessKeyData).map(addUserInfoToAccessKeys(_, users))
+  }
+
+  def addUserInfoToAccessKeys(accessKeyData: List[AccessKeyMetadata], users: Seq[VulnerableUser]): List[VulnerableAccessKey] = {
+    addAccessKeyIds(users.filter(_.humanUser), accessKeyData).toList ++
+      addAccessKeyIds(users.filterNot(_.humanUser), accessKeyData)
```

Comment:
I think this can just be `addAccessKeyIds(users, accessKeyData)` now, which probably means we just call `addAccessKeyIds` directly and remove this method.